### ISSUE TITLE
Error processing helptags

### ISSUE THREAD

**Issue by williamboman** [yardnsm/vim-import-cost#14]
When running `:PluginInstall` with [Vundle.vim](https://github.com/VundleVim/Vundle.vim) I run into the following issue with `vim-import-cost`:

```
[2018-08-26 19:30:37] > Error running :helptags /Users/william.boman/.vim/bundle/vim-import-cost/doc
```

**Comment by yardnsm** [comment]
Thanks for the report! The error was caused due to duplicated helptag, fixed in ff166ec974a36640632024cc48c07d5ff092d2db.

**Comment by williamboman** [comment]
Awesome, thanks!
